docs/20-package-management.md

```diff
@@ -4,7 +4,19 @@ The doAzureParallel package allows you to install packages to your pool in two w
 - Installing on pool creation
 - Installing per-*foreach* loop
 
+Packages installed at the pool level benefit from only needing to be installed once per node. Each iteration of the foreach can load the library without needing to install them again. Packages installed in the foreach benefit from specifying any specific dependencies required only for that instance of the loop.
```

Comment:
"benefit from specifying any specific" is a weird sentence. May be "benefit from specifying any dependencies"

Reply:
fixed.